src/components/common/CardContent/CardContentStyled.tsx

```diff
@@ -67,9 +69,11 @@
   font-weight: 400;
   line-height: normal;
   background-color: #ffffff;
-}
 
-.popover button:hover {
-  background-color: var(--gray10);
-  color: var(--primary);
-}
+  & hover {
+    background-color: var(--gray10);
+    color: var(--primary);
+  }
```

Comment:
```suggestion
  &:hover {
    background-color: var(--gray10);
    color: var(--primary);
  }
```
위처럼 되어야 할 것 같은데, 혹시 동작 확인해 보셨나요?